internal/service/advancedcluster/resource_advanced_cluster_migration_test.go

```diff
@@ -113,11 +112,6 @@ func TestMigAdvancedCluster_shardedMigrationFromOldToNewSchema(t *testing.T) {
 				Config:            configShardedTransitionOldToNewSchema(orgID, projectName, clusterName, false),
 				Check:             checkShardedTransitionOldToNewSchema(false),
 			},
-			{
```

Comment:
nit: We can also simplify code in CheckClusterExistsHandlingRetry as it is no longer necessary to retry